Tests/ProGetAutomationTest/ProGetAutomationTest.psm1

```diff
@@ -0,0 +1,78 @@
+
+$apiKey = 'HKgaAKWjjgB9YRrTbTpHzw=='
+$credential = New-Credential -UserName 'Admin' -Password 'Admin'
+
+$pgNotInstalledMsg = 'It looks like ProGet isn''t installed. Please run init.ps1 to install and configure a local ProGet instance so we can run automated tests against it.'
+$svcRoot = Get-ItemProperty -Path 'hklm:\SOFTWARE\Inedo\ProGet' -Name 'ServicePath' | Select-Object -ExpandProperty 'ServicePath'
+if( -not $svcRoot )
+{
+    throw $pgNotInstalledMsg

```

Comment:
This is an appropriate time to throw the error. We want to immediately terminate all tests if ProGet isn't installed.